src/vivarium_census_prl_synth_pop/tools/cli.py

```diff
@@ -308,13 +289,37 @@ def jobmon_make_results_runner(
     "abbreviation.",
 )
 def make_state_results(
-    processed_results_dir: Path,
+    processed_results_dir: str,
     verbose: int,
     with_debugger: bool,
     state: str,
 ) -> None:
+    """Subset the processed results found in PROCESSED_RESULTS_DIR to a specific STATE.
+    \f
+    :param processed_results_dir: Directory where the already-processed results exist.
+    :param verbose: Configure logging verbosity.
+    :param with_debugger: Drop into python debugger if an error occurs.
+    :param state: The state to subset to. This should be the two letter postal abbreviation.
+    """
+    resolved_results_dir = Path(processed_results_dir).resolve()
+    # Ensure only a single expected results directory exists
+    resolved_results_dir = [
```

Comment:
it was super confusing which path had to be passed in for this to work, so now it basically doesn't matter as long as there is one and only one "pseudopeople_input_data_usa" directory nested somewhere.

Reply:
So much better